Resources/config/security.xml

```diff
@@ -10,6 +10,13 @@
     </parameters>
     <services>
         <service id="sonata.admin.security.handler.noop" class="%sonata.admin.security.handler.noop.class%" public="false"/>
+        <service id="sonata.admin.security.handler.cache"
+            class="Sonata\AdminBundle\Security\Handler\CacheSecurityHandler"
+            decorates="sonata.admin.security.handler"
```

Comment:
Argh. Yes, sure. But please add `NEXT_MAJOR` instructions explaining how to switch to native decoration